tests/installation/livecdreboot.pm

```diff
@@ -81,9 +82,10 @@ sub run() {
             save_screenshot;
         }
         if (get_var("XEN")) {
+            send_key 'up';    # prevent grub2 timeout
```

Comment:
It was just a question. I don't know if this is a good decision. I recommend to give PRs a bit more time for others to review. "+1" by me means "Looks good to me but others must approve".